### PR TITLE
Cross linking for Forms subjects

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Still working on it

There are lots of documentation within the docs regarding forms. 

Some are redundant, some are oriented for module developers, other for core contributors, ... i'm trying to arrange those pages. 

Still WIP

### CODE REVIEW

**Review by kpodemski — COMMENTED** [review]
I'm willing to accept and merge this PR the remaining thing would be to create an issue and explain what's left to do here or how could we make it better :)

**Review by tleon — APPROVED** [review]

**Comment by thomasnares** [comment]
This one could be merged @kpodemski, issue is here: #1765

File: development/architecture/migration-guide/forms/CRUD-forms.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ weight: 20
...
+{{% notice tip %}}
+If you are looking to customize an identifiable object with a module, please have a look at [Grid and identifiable object form hooks usage example]({{<relref "/8/modules/sample-modules/grid-and-identifiable-object-form-hooks-usage">}}).
+{{% /notice %}}
+
+{{% notice tip %}}
+If you are looking for the Form component documentation, please have a look at [Forms with Symfony]({{<relref "/8/development/components/form">}}).
+{{% /notice %}}
+
+{{% notice tip %}}
+This guide is intented for core developers looking to understand how PrestaShop now handles Forms in the core.
```
Comment (kpodemski) [inline development/architecture/migration-guide/forms/CRUD-forms.md]:
```suggestion
This guide is intended for core developers and contributors who want to understand how PrestaShop handles forms in the core.
```

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @thomasnares

**Pull request merged**